doc/docs/modules/ROOT/pages/faq.adoc

```diff
@@ -56,3 +56,8 @@ all the values for that field will be casted to String. This because Spark is no
 and need each column to always have the same type.
 
 You can read more <<quickstart.adoc#bookmark-read-known-problem, here>>.
+
+== The returned columns are not in the same ordere as I specified in the query
+
+Unfortunately this is a known issue and is there for Neo4j 3.* and Neo4j 4.0.
+With Neo4j 4.1+ you will get the same order as specified in the return statement.
```

Comment:
@moxious please note this

Reply:
Understood!  And it's perfect to put it into a FAQ like this.   But since you are drawing my attention, "order" doesn't have an extra e at the end of the word.  :)